docs/docs/contributors.md

```diff
@@ -20,6 +20,14 @@ cd rise-tools
 npm install
 ```
 
+## Generate Build
+
+After installing the dependencies, you need to generate the build for the project if you want to run the playground or the demo server:
+
+```sh
+npm run build
+```
```

Comment:
Would be great to mention that its best to run `npm run watch` afterwards to continue seeing changes while working on server-related things.

Here's some context:
- We generally use `tsconfig.json` paths to rewrite paths from `lib` to `src`. That way, in development, we can simply load source code without having to start compilation. If you're running the React Native application (Playground) or running tests with Vitest, they both get raw TypeScript and compile it on the go.
- Unfortunately, `tsx` (that we use to run the demo server) does not recognise paths, and as a result, will always fallback to loading `lib`. 

TL;DR 

Easy version: **always keep npm run watch in the background**

Reply:
got it. 

changed the build command to watch and moved the info into the 'Run Demo Server section' as it's required only there. I will see why the tsx is not working even though its explicitly provied as `tsx watch ./src`.